util/dvsim/SimResults.py

```diff
@@ -84,7 +85,7 @@ def __init__(self, items, results):
     def _add_item(self, item, results):
         '''Recursively add a single item to the table of results'''
         status = results[item]
-        if status == "F":
+        if status in ["F", "K"]:
```

Comment:
@matutem, this logic allows the bucketization of killed jobs. Jobs killed due to timeout (and all other reasons) now also show up in the report.